cdap-runtime-ext-dataproc/src/main/resources/gcp-dataproc.json

```diff
@@ -32,6 +32,15 @@
     {
       "label": "General Settings",
       "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Network Host Project ID",
```

Comment:
It's kind of weird to have this first, as it might confuse people who don't need it.

Can we move 'Network' below 'Zone', and put this below 'Network'?

Reply:
Done.